modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C

```diff
@@ -79,6 +79,10 @@ INSFVRhieChowInterpolator::validParams()
       "supplied when the mesh dimension is greater than 2. It represents the on-diagonal "
       "coefficients for the 'z' component velocity, solved "
       "via the Navier-Stokes equations.");
+  params.addParam<NonlinearSystemName>("mass_momentum_system",
+                                       "0",
```

Comment:
Is this the default name that you want? The default nonlinear system name for our standard monolithic solves is `"nl0"`

Reply:
I just think it's a bit odd that the default name is a number

Reply:
I think I think `_fe_problem.nlSysNum` tries to parse it to a number and if it can it will select the number. I'll switch to the default you mentioned! Commit incoming.